packages/core/frontend/sirius-components-core/src/icon/IconOverlay.tsx

```diff
@@ -40,13 +26,12 @@ export const IconOverlay = ({
 }: IconOverlayProps) => {
   const iconWidth: number = customIconWidth || 16;
   const iconHeight: number = customIconHeight || 16;
-  const classes = useStyles({ iconHeight, iconWidth });
   const { httpOrigin } = useContext<ServerContextValue>(ServerContext);
 
   return (
     <>
       {iconURL?.length > 0 && (
-        <div className={classes.iconContainer} style={{ ...customIconStyle }}>
+        <div style={{ ...customIconStyle, position: 'relative', width: iconWidth, height: iconHeight }}>
```

Comment:
Not sure if we get the same behavior as before here, with `customIconStyle` placed *before* the explicit attributes.

Reply:
done